### PR TITLE
Fix script

### PR DESCRIPTION
**Pull request by Aleksandriys**

### CODE REVIEW

**Comment by Aleksandriys** [comment]
Ready

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 1]
```diff
@@ -1,25 +1,138 @@
...
+    {
+      os = "Android",
+      carrier = "Megafon",
+      firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+      osVersion = "4.4.2",
+      maxNumberRFCOMMPorts = 1
+    }
+  }
+
+local function Backup_preloaded()
```
Comment (VVeremjova) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys I have seen common functions for backup/restore file. Use them

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 2]
```diff
@@ -1,25 +1,129 @@
...
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test.Precondition_StopSDL()
+  StopSDL()
+end
+
+function Test.Precondition_DeleteLogsAndPolicyTable()
+commonSteps:DeleteLogsFileAndPolicyTable()
```
Comment (VVeremjova) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys indents

File: test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua [hunk 3]
```diff
@@ -1,25 +1,129 @@
...
+  self:connectMobile()
+end
+
+function Test:TestStep_CreateSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+end
+
+function Test:TestStep_RegisterApplication()
+ local corId = self.mobileSession:SendRPC("RegisterAppInterface", registerAppInterfaceParams)
```
Comment (VVeremjova) [inline test_scripts/Polices/Policies_Security/003_ATF_P_PolicyTable_Certificate_EMPTY_value.lua]: @Aleksandriys indents

**Pull request merged**